api/src/test/java/org/openmrs/module/openconceptlab/ImportServiceTest.java

```diff
@@ -311,7 +311,9 @@ public void run_shouldSetSubscriptionUrl() {
 		importer.setSaver(saver);
 
 		TestResources.setupDaemonToken();
-		importer.run(TestResources.getSimpleResponseAsJsonStream());
+		InputStream in = TestResources.getSimpleResponseAsJsonStream();
+		importer.setInputStream(in);
+		importer.run(in);

```

Comment:
@rkorytkowski This one was caused by useless conditionals that you've mentioned before. Removed.